src/main/java/utils/SeleniumHelper.java

```diff
@@ -104,5 +103,14 @@ public static void replaceElementValue(WebElement element, String value)
         }
         element.sendKeys(value);
     }
+    public static boolean verifyElementPresent(String locator) {
+
+        try {
+            SeleniumDriver.getDriver().findElement(By.xpath(locator));
+            return true;
+        } catch (Throwable e) {
```

Comment:
Don't catch `Throwable`. It's rarely needed. Catch specific Exception(s) or `Exception`.

Reply:
Caught exception.